packages/scandipwa/src/query/ProductList.query.js

```diff
@@ -921,19 +921,19 @@ export class ProductListQuery {
     }
 
     _getBaseFinalPriceExclTaxField() {
-        return new Field('base_final_price_excl_tax')
+        return new Field('default_final_price_excl_tax')
             .addField('currency')
             .addField('value');
     }
 
     _getBasePriceField() {
-        return new Field('base_price')
+        return new Field('default_price')
             .addField('currency')
             .addField('value');
     }
 
     _getBaseFinalPriceField() {
-        return new Field('base_final_price')
+        return new Field('default_final_price')
             .addField('currency')
             .addField('value');
     }
```

Comment:
In the new variant, the methods' names don't respect actual Fields that are being requested. 
Please make the methods' names consistent with the requestable fields.

Reply:
Also: Please make sure that this change is 100% necessary. This change could cause some trouble when migrating existing projects to the new version, due to the changed variable names.